modules/local/nf-core-modified/mafft.nf

```diff
@@ -9,26 +12,27 @@ process MAFFT {
         'quay.io/biocontainers/mafft:7.490--h779adbc_0' }"
 
     input:
-    tuple val(meta), path(fasta)
+    path(fasta)
 
     output:
-    tuple val(meta), path("*-mafft.fa"), emit: msas
+    path("*-mafft.fa") , emit: msas
     path "versions.yml"           , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.og}"
     """
+    prefix=\$(basename "${fasta}" .fa)
+
     mafft \\
         --thread ${task.cpus} \\
         --localpair \\
         --maxiterate 1000 \\
         --anysymbol \\
         ${fasta} \\
-        > ${prefix}-mafft.fa
+        > \$prefix-mafft.fa
```

Comment:
why do the curly braces break here? or rather why did you remove them?

Reply:
One refers to a bash variable (new), the other refers to a groovy variable (old)